src/features/quick_reblog.js

```diff
@@ -316,6 +316,7 @@ const preventLongPressMenu = ({ originalEvent: event }) => {
 };
 
 export const main = async function () {
+  if (!userBlogs.find(({ primary }) => primary === true)) return;
```

Comment:
Let's DRY this

https://github.com/AprilSylph/XKit-Rewritten/blob/317a3de992b53c323808fb8296daadccd9a253b2/src/utils/user.js#L32-L35

```suggestion
  if (!primaryBlog) return;

```

Reply:
Oh yeah! Totally forgot we had that.